### DISCUSSION TITLE
About material.yaml

### DISCUSSION THREAD

**Discussion by FURUNZ** [SeisSol/SeisSol#1091] [Q&A]
Take point source-LOH 1 as an example, his material parameter file
! Any
components:
-! GroupFilter
groups: 1
components: ! ConstantMap
map:
rho: 2600.0
mu: 1.04e10
lambda: 2.08e10
-! GroupFilter
groups: 2
components: ! ConstantMap
map:
rho: 2700.0
mu: 3.23980992e10
lambda: 3.24038016e10
groups1 and groups2 here are specific materials for which, if I want to set personalized parameters according to the formation lithology studied by myself, how can I achieve it? This example is stratified formation consisting of two layers, is groups1 and groups2 here from top to bottom by default

**Comment by Thomas-Ulrich** [comment]
what kind of personalized parameters you want to set? Elastic parameters?
Are we talking about depth-dependent material properties? Or curved interfaces?
